tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -699,23 +715,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! grep OBJECTIVE example_model/*pdb >> example_model/scores.txt"
+    "! grep OBJECTIVE example_models/*pdb >> example_models/scores.txt"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- Open this file in vim and type:\n",
-    "\n",
-    "\t`Shift :` to enter command input\n",
-    "    \n",
+    "- Open this file in vim and type:    \n",
```

Comment:
As per above's vim removal, would is be safer to avoid vim altogether?

We could easily do:
```bash
sort -rnk6 <file> | head -n 10
```

Reply:
I think pedagogically its sometimes useful to have interactive editing